Week-1/Homework/2.DanishLetters.js

```diff
@@ -0,0 +1,33 @@
+//find and count danish letters
+
+function danishLetters(string) {
+  //return errrors
+  if (typeof string != "string") {
+    return;
+  }
+
+  string.toLowerCase();
+
+  let output = { total: 0 };
+
+  if (string.match("æ") !== null) {
```

Comment:
Can you think of a way to do this without using regex?

Reply:
ez solved